flink-runtime/src/main/java/org/apache/flink/runtime/state/DefaultOperatorStateBackend.java

```diff
@@ -106,6 +112,8 @@ public void dispose() {
 
 		Preconditions.checkNotNull(stateDescriptor);
 
+		stateDescriptor.initializeSerializerUnlessSet(getExecutionConfig());
```

Comment:
Got your meaning, thanks a lot for your review.  the suggesting are made sense, but I want to keep my point of view and then let the community people give advice.